go.mod

```diff
@@ -4,6 +4,8 @@ go 1.11
 
 require (
 	github.com/BurntSushi/toml v0.3.1
+	github.com/andreyvit/diff v0.0.0-20170406064948-c7f18ee00883
 	github.com/cpuguy83/go-md2man/v2 v2.0.0-20190314233015-f79a8a8ca69d
-	gopkg.in/yaml.v2 v2.2.2
+	github.com/sergi/go-diff v1.1.0 // indirect
```

Comment:
I hate to be a broken record, but please remove this change 🙏 I would rather this PR not add new requirements, even if they are only for testing.

Reply:
No problem- done.